.env.example

```
@@ -92,6 +93,18 @@ SFTP_PRIVATE_KEY=
 SFTP_PASSPHRASE=
 
 
+# WebDAV settings. Required if you're using a WebDAV server (e.g. NextCloud, Owncloud)
+# to store your media (STORAGE_DRIVER=webdav).
+# For NextCloud, the base URI looks like https://your-nextcloud.example/remote.php/dav/files/<username>/
+WEBDAV_BASE_URL=
+WEBDAV_USERNAME=
+WEBDAV_PASSWORD=
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The WEBDAV_PASSWORD key should go before the WEBDAV_USERNAME key

(UnorderedKey)